### PR TITLE
feat(judicial-system): Handle case where defendants do not have a national id

### PR DESCRIPTION
**Pull request by oddsson**
# Handle case where defendants do not have a national id

https://app.asana.com/0/1199153462262248/1201337986358804/f

## What

Handle case where defendants do not have a national id

## Why

Defendants are often foreigners who do not have an Icelandic national id.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/migrations/20220204105457-update-defendant.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      Promise.all([
+        queryInterface.addColumn(
+          'defendant',
+          'no_national_id',
+          {
+            type: Sequelize.STRING,
```
Comment (oddsson) [inline apps/judicial-system/backend/migrations/20220204105457-update-defendant.js:11]: Are booleans represented as strings in the backend?

Comment (gudjong) [reply apps/judicial-system/backend/migrations/20220204105457-update-defendant.js:11]: absolutely not

File: apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ export const StepOne: React.FC = () => {
           name: theCase.defendants[0].name,
           address: theCase.defendants[0].address,
           nationalId: theCase.defendants[0].nationalId,
+          noNationalId: theCase.defendants[0].noNationalId,
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx:49]: Shouldn't citizenship be updated as well?

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx (hunk 2 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Prosecutor/RestrictionRequest/StepOne/StepOne.tsx:49]: Jup! Thanks 🙏

**Review by gudjong — APPROVED** [review]

**Pull request merged**